### DISCUSSION TITLE
防刷验证 无法显示验证码

### DISCUSSION THREAD

**Discussion by kinguler** [Licoy/wordpress-theme-puock#112] [Q&A]
开启防刷验证无法显示验证码，如何破？验证码是无法显示图片，我也装了 Mimi Captcha 验证码插件都开启了。
还有就是 ，留言板页面是如何创建？谢谢。

**Comment by Licoy** [comment]
1. 验证码无法显示请直接访问那个链接查看错误信息
2. 留言板直接创建一个新页面，开启允许评论即可

**Comment by kinguler** [comment]
![image](https://user-images.githubusercontent.com/33061007/150716540-720ac3d8-3fa9-48d9-8a58-042f1f9d2a30.png)
所以页面下面评论都是这样

**Comment by Licoy** [comment]
请先查看错误信息，你直接贴图无法解决，另外需要开启`gd2`扩展

**Comment by kinguler** [comment]
![image](https://user-images.githubusercontent.com/33061007/150717031-7c9931f0-059b-4740-b5b5-9455b2f177b5.png)
后台是可以显示验证码的，您说的错误信息咋看？求指教

**Comment by Licoy** [comment]
![image](https://user-images.githubusercontent.com/20410697/150717167-db12ed10-f79b-4fa5-a584-84ef3279020d.png)

右键->复制图片链接->新窗口打开， 然后查看页面错误信息

**Comment by kinguler** [comment]
此站点遇到了致命错误。

了解有关对WordPress进行故障排除的更多信息。

**Comment by Licoy** [comment]
可能是没开启gd2扩展导致的，或者查阅[错误查看办法（第三方）](https://www.wpyi.com/zhiming-cuowu.html)来进行查看错误

**Comment by kinguler** [comment]
求教如何开启gd2扩展,

**Comment by Licoy** [comment]
这个百度一下就有

**Comment by kinguler** [comment]
找到php的配置文件php.ini，搜索extension=php_gd2.dll，去掉前面的分号即可；
如果没有直接添加这种情况适合于windows系统和编译时支持gd的php，保存后重启apache即可
如果用的是安装版本如Ubuntu的deb。redhat的rpm安装的php可以使用命令安装即可
Ubuntu:

**Comment by kinguler** [comment]
这个php.ini 是服务器上的？还是WordPress？ 可是服务器是控制面板，是虚拟主机

**Comment by kinguler** [comment]
控制面板里并没有 PHP 扩展，是不是就无法开启 gd2

**Comment by Licoy** [comment]
如果是自己的服务器可以手动开启，如果是虚拟主机面板就咨询一下主机商吧

**Comment by kinguler** [comment]
![image](https://user-images.githubusercontent.com/33061007/150722450-15652251-e012-4b5f-9d24-057a456404f5.png)
会是这个文件吗？

**Comment by Licoy** [comment]
问题不是靠猜的，如果是你图中的问题你整个站点就进不去，先确认gd2扩展是否开启再说吧

**Comment by kinguler** [comment]
好的，谢谢。

**Comment by kinguler** [comment]
问题解决了，服务商给换成php7.4。之前我选的php8.0。再次感谢
